js/ui/grid_core/ui.grid_core.editing_form_based.js

```diff
@@ -57,6 +57,14 @@ export const editingFormBasedModule = {
                     this._updateEditFormDeferred = null;
 
                     this.callBase.apply(this, arguments);
+
+                    this._rowsView.renderCompleted.add(() => {
+                        const rowIndex = this.getEditRowIndex();
+
+                        if(rowIndex !== -1 && this.isEditing() && this.isPopupEditMode()) {
```

Comment:
you also need to add flag like `isFirstRender` to prevent calling `_showEditPopup` in case editRow is changed